frontend/src/components/search/SearchOpportunityStatus.tsx

```diff
@@ -26,6 +27,10 @@ export default function SearchOpportunityStatus({
   const handleCheck = (value: string, isChecked: boolean) => {
     const updated = new Set(query);
     isChecked ? updated.add(value) : updated.delete(value);
+    // Add "status=NO_STATUS_VALUE" if no values are selected.
```

Comment:
[nit] not really related to this change, but since this is a client side component, we probably want to wrap this function in a useCallback to limit re-renders